tests/longevity-48h-verify-multiDC.yaml

```diff
@@ -1,47 +1,48 @@
+prepare_verify_cmd: "cassandra-stress read  cl=ALL n=200200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..200200300"]
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=2860m -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=00200300..600200300"]
+stress_read_cmd: ["cassandra-stress read cl=QUORUM duration=100m -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..200200300"]
 cassandra_stress_duration: 10080
 cassandra_stress_threads: 1000
 cassandra_stress_population_size: 100000000000
-#n_db_nodes: 3
-n_db_nodes: '5 5 3'
-n_loaders: 3
+n_db_nodes: '3 3'
+n_loaders: 1
 n_monitor_nodes: 1
-nemesis_class_name: 'MdcChaosMonkey'
-#nemesis_class_name: 'DrainerMonkey'
+nemesis_class_name: 'NoOpMonkey'
 nemesis_interval: 5
-user_prefix: 'sct-multidc'
+user_prefix: 'yaron-2-3-0-multidc'
 failure_post_behavior: keep
 space_node_threshold: 6442
 ip_ssh_connections: 'public'
 
-ami_id_db_scylla_desc: '1.7.rc2'
+ami_id_db_scylla_desc: '2-3-0'
+
+use_mgmt: true
+mgmt_port: 10090
+scylla_repo_m: 'http://repositories.scylladb.com/scylla/repo/7b02fff5-e4d0-4e4d-ad12-e605ca4873c2/centos/scylladb-2018.1.repo'
+scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/branch-1.2/44/scylla-manager.repo'
 
-#es_url:
-#es_user:
-#es_password:
 
 backends: !mux
     aws: !mux
         # What is the backend that the suite will use to get machines from.
         cluster_backend: 'aws'
         # From 0.19 on, iotune will require bigger disk, so let's use a big
         # loader instance by default.
-        instance_type_loader: 'c3.large'
+        instance_type_loader: 'c4.large'
         # Size of AWS monitor instance
-        instance_type_monitor: t2.small
+        instance_type_monitor: i3.large
         us_east_1_and_us_west_2:
-            user_credentials_path: ''
+            user_credentials_path: '~/.ssh/scylla-qa-ec2'
             region_name: 'us-east-1 us-west-2'
             security_group_ids: 'sg-c5e1f7a0 sg-81703ae4'
             subnet_id: 'subnet-ec4a72c4 subnet-5207ee37'
-            ami_id_db_scylla: 'ami-40a0e756 ami-77e78617'
+            ami_id_db_scylla: 'ami-0fc423ac17a75570d ami-0da599147b1d9e80d'
             ami_db_scylla_user: 'centos'
-            ami_id_loader: 'ami-40a0e756'
+            ami_id_loader: 'ami-0fc423ac17a75570d'
             ami_loader_user: 'centos'
-            ami_id_db_cassandra: 'ami-ada2b6c4'
-            ami_db_cassandra_user: 'ubuntu'
-            ami_id_monitor: 'ami-40a0e756'
+            ami_id_monitor: 'ami-010f2b2749b78a6c5'
```

Comment:
Change back all AMI-ID as it was in previous version of your commit.

Reply:
done